### ISSUE TITLE
Locale 'C' fails install

### ISSUE THREAD

**Issue by rrva** [ViDA-NYU/reprozip#102]
if I have locale "C" install fails. Setting locale to en_US.utf8 works.
```
[root@localhost ~]# pip install reprozip
Collecting reprozip
  Downloading reprozip-0.6.1.tar.gz
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-yxg7ewqv/reprozip/setup.py", line 27, in <module>
        description = fp.read()
      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1286: ordinal not in range(128)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

      File "<string>", line 20, in <module>

      File "/tmp/pip-build-yxg7ewqv/reprozip/setup.py", line 27, in <module>

        description = fp.read()

      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode

        return codecs.ascii_decode(input, self.errors)[0]

    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1286: ordinal not in range(128)

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-yxg7ewqv/reprozip
```

**Comment by remram44** [comment]
Thanks for the report!

This look like a pretty stupid behavior from Python 3, I'll see what I can do. It seems that my firstname is the issue, as it contains the only non-ascii character in the README.

**Comment by remram44** [comment]
Fixed by 2f0ed486.

I can't reproduce this on Travis (https://travis-ci.org/ViDA-NYU/reprozip/jobs/52673849#L156) but it happens locally -- what are they doing over there?

#python:
> 17:31 < Remram> PY3 seems to use the locale
> 17:31 < Remram> so depending on the system's locale, 'pip install' will work or crash
> 17:32 < _habnabit> Remram, that's... really awful
> 17:32 < _habnabit> Remram, i'm verifying locally though
> 17:34 < _habnabit> Remram, wow. that's really bad
> 17:34 < _habnabit> Remram, well, you can specify the file encoding as a parameter to open
> 17:34 < _habnabit> Remram, for compatibility, you'd have to use io.open
> 17:34 < Remram> _habnabit: yes I was going to do that
> 17:34 < _habnabit> Remram, well, that'll fix it then
> 17:35 < _habnabit> Remram, but holy crap that's really awful
> 17:35 < _habnabit> Remram, i thought python 3 was better than that
> 17:35 < Remram> yes, Python 3's unicode handling is a lot worse than PY2
> 17:35 < _habnabit> Remram, no disagreement here
